flocker/docs/bootstrap/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
```

Comment:
Some docstrings missing from various classes and functions in modules in this package.